### PR TITLE
Best practices migration

### PR DESCRIPTION
**Pull request by idosun**
- Add the new migration guide
- Rename Grouping & Fingerprints guide

### CODE REVIEW

File: src/collections/_documentation/guides/grouping-and-fingerprints/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Grouping & Fingerprints
+title: Customize Event Grouping
```
Comment (BYK) [inline src/collections/_documentation/guides/grouping-and-fingerprints/index.md]: This seems like an unrelated change.

File: src/collections/_documentation/guides/integrate-frontend/upload-source-maps.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ sidebar_order: 4
...
-Releases are used for applying `source maps` to minified JavaScript to view original, untransformed source code in the stack traces of errors associated with the respective release. This is particularly useful for debugging minified code (for example, UglifyJS), or transpiled code from a higher-level language (for example, TypeScript, ES6).
 
  In this tutorial, we will:
 
  1. Utilize the `Sentry Command Line Interface` (CLI) **during the build process** to update your Sentry account by:
     - Creating a new release version
     - Uploading the project's latest source maps (and associate them with the new release version)
-2. Add the release version to the Sentry SDK configuration --- this associates any error captured by the SDK in our app to this specific release. Sentry uses the release's source maps we uploaded to source map the event's stack trace.
+
+2. Add the release version to the Sentry SDK configuration --- this will associate any error captured by the SDK in our app to this specific release. Sentry will use the release's source maps we uploaded to unminify the error's stack trace.
```
Comment (BYK) [inline src/collections/_documentation/guides/integrate-frontend/upload-source-maps.md]: Moar unrelated changes?

File: src/collections/_documentation/guides/migration/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: Moving on Up - Switch to Hosted Sentry
+sidebar_order: 6
+---
+
+Sentry offers a hosted cloud solution in addition to an open-source one, both are functionally the same. Despite open-source being a core value for us at Sentry, it isn't necessarily recommended for everyone. As Sentry evolves, many customers are finding out that hosting Sentry on-premise can quickly become expensive to maintain, scale, and support, making our SaaS product, the better and less costly option.
+
+For more information take a look at our post on [Self Hosted or Cloud Sentry?](https://sentry.io/_/resources/about-sentry/self-hosted-vs-cloud/)
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]: Just read this: https://xscode.com/686/open-source-is-free-but-expensive/

Might be worth referencing it too as this is written by an independent party. We can also refer to this from the "self-hosted vs cloud" page.

File: src/collections/_documentation/guides/migration/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+### 2. Choose a plan
+
+Take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Notice that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+
+If you're expecting higher volumes or interested in our Enterprise capabilities (priority support and SLAs, single-tenant options, etc.), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
+
+### 3. Export your data
+
+Sentry open-source installs a [Command-Line](https://docs.sentry.io/server/cli/) script that allows you to perform various operations that are unachievable within the web UI. One of those is `export`, exporting your data into a transport JSON.
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]: Since you are only referring to the Docker-based command, it may make more sense to say the following:
```suggestion
Sentry open-source provides a [command line interface](https://docs.sentry.io/server/cli/) that allows you to perform various operations that are unachievable within the web UI. One of those is `export`, exporting your data into a transport JSON.
```

File: src/collections/_documentation/guides/migration/index.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Run the following command in your terminal to start the export script and redirect the output (_containing_ the transport JSON) to a file:
+
+    $ docker-compose run --rm web export > sentry_export.txt
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
    $ docker-compose run --rm web export > sentry_export.json
```

File: src/collections/_documentation/guides/migration/index.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    
+> This invokes the sentry CLI `export` command on a new isolated instance of our Docker web container.
+
+### 4. Contact us
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
### 4. Ship it to us
```

File: src/collections/_documentation/guides/migration/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+To kick-off the migration process shoot us an email with the following details at <support@sentry.io>:
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
To kick-off the migration process, shoot us an email at <support@sentry.io> with the following details:
```

File: src/collections/_documentation/guides/migration/index.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+- Email Subject: `Self-Hosted Migration Application`
+- Your **organization slug**
+- Your **selected plan**. If you're unsure, let us know. We'll be happy to help you make the right decision. Also, if you think you'll need an Enterprise plan we can connect you with the right person to discuss that plan.
+- Attach the **export output** containing the transport JSON.
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
- Attach or link to the **sentry_export.json** file.
```

File: src/collections/_documentation/guides/migration/index.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+Alternatively, you can also reach out to us through [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/)
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]: Maybe we should promote this over the e-mail as reading those steps make me cringe a bit (feels cumbersome, especially the way we ask people to share the export file with us).

File: src/collections/_documentation/guides/migration/index.md (hunk 5 above)
Comment (idosun) [reply src/collections/_documentation/guides/migration/index.md]: I changed it to *.txt as the output contains the JSON object + log outputs...

File: src/collections/_documentation/guides/grouping-and-fingerprints/index.md (hunk 1 above)
Comment (idosun) [reply src/collections/_documentation/guides/grouping-and-fingerprints/index.md]: yes..

File: src/collections/_documentation/guides/integrate-frontend/upload-source-maps.md (hunk 2 above)
Comment (idosun) [reply src/collections/_documentation/guides/integrate-frontend/upload-source-maps.md]: yes..

File: src/collections/_documentation/guides/migration/index.md (hunk 5 above)
Comment (BYK) [reply src/collections/_documentation/guides/migration/index.md]: I'll try to fix those log lines. It should still be `json` I'd say. @mattrobenolt?

File: src/collections/_documentation/guides/grouping-and-fingerprints/index.md (hunk 1 above)
Comment (BYK) [reply src/collections/_documentation/guides/grouping-and-fingerprints/index.md]: We should not bundle unrelated changes, that's why I highlighted this one :)

File: src/collections/_documentation/guides/migration/index.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+Run the following command in your terminal to start the export script and redirect the output (_containing_ the transport JSON) to a file:
+
+    $ docker-compose run --rm web export > sentry_export.json
+    
+> This invokes the sentry CLI `export` command on a new isolated instance of our Docker web container.
+
+### 4. Ship it to us
+
+To kick-off the migration process contact to us through [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/). Once we get your request, one of our support reps will reach out though a support ticket to get the following details:
```
Comment (BYK) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
To kick-off the migration process, contact us through [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/). Once we get your request, one of our support reps will reach out through a support ticket to get the following details:
```

Comment (BYK) [reply src/collections/_documentation/guides/migration/index.md]: Requested changes for the small typo here: tough -> through

File: src/collections/_documentation/guides/migration/index.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+![Org Slug]({% asset guides/migration/02.png @path %})
+
+> Note: You can modify your org slug in the org settings.
+
+### 2. Choose a plan
+
+Take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Notice that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+
+If you're expecting higher volumes or interested in our Enterprise capabilities (priority support and SLAs, single-tenant options, etc.), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
```
Comment (MimiDumpling) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, single-tenant options, etc.), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
```

File: src/collections/_documentation/guides/migration/index.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+    $ docker-compose run --rm web export > sentry_export.json
+    
+> This invokes the sentry CLI `export` command on a new isolated instance of our Docker web container.
+
+### 4. Ship it to us
+
+To kick-off the migration process contact to us through [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/). Once we get your request, one of our support reps will reach out though a support ticket to get the following details:
+
+- Your **Organization slug**.
+- **Selected plan**. If you're unsure, let us know. We'll be happy to help you make the right decision. Also, if you think you'll need an Enterprise plan we can connect you with the right person to discuss that plan.
```
Comment (MimiDumpling) [inline src/collections/_documentation/guides/migration/index.md]:
```suggestion
- **Selected plan**. If you're unsure, let us know. We'll be happy to help you make the right decision. Also, if you think you'll need an Enterprise plan, we can connect you with the right person to discuss details.
```

File: src/collections/_documentation/guides/migration/index.md (hunk 4 above)
Comment (BYK) [reply src/collections/_documentation/guides/migration/index.md]: I think "installs" was not accurate here as there won't be a `sentry` command with the Docker-based install as we discussed earlier.
